DopplerJobsServer/appsettings.qa.json

```diff
@@ -11,14 +11,16 @@
   },
   "Jobs": {
     "DopplerBillingJobSettings": {
-      "IntervalCronExpression": "0 0 1 12 *",
+      "IntervalCronExpression": "0 8 1 * *",
       "Identifier": "Doppler_Billing_Job",
       "StoredProcedures": []
     },
     "DopplerCurrencyJob": {
-      "IntervalCronExpression": "0 0 1 12 *",
+      "IntervalCronExpression": "0 17 * * mon-fri",
       "Identifier": "Doppler_Currency_Job"
-    }
+    },
+    "CurrencyEndpoint": "https://apisqa.fromdoppler.net/sap-test/Billing/SetCurrencyRate",
+    "BillingEndpoint": "https://apisqa.fromdoppler.net/sap-test/Billing/CreateBillingRequest"
```

Comment:
should it point to `https://apisqa.fromdoppler.net/sap-test` or to `https://apis.fromdoppler.com/sap-test`?

Reply:
`https://apis.fromdoppler.com/sap-test ` I gonna change it